### PR TITLE
Book withdraw from classes

### LINKED ISSUE #35: Add booking functionality to classes page
- Each class displayed has a "Book Now" button 
- When this button is clicked, the backend checks whether the user is logged in or not
- If the used is logged in, create a new record in the userClasses table
- If the user is not logged in, redirect them to the login page

### PR DESCRIPTION
**Pull request by kvlin**

### CODE REVIEW

File: routes/html-routes.js [hunk 1]
```diff
@@ -32,12 +32,6 @@ module.exports = function(app) {
     });
   });
   app.get("/classes", (req, res) => {
-    let loggedIn = false;
```
Comment (jkaho) [inline routes/html-routes.js]: Just add this stuff back in - since it has to do with the login/logout button

File: routes/html-routes.js [hunk 2]
```diff
@@ -57,7 +51,7 @@ module.exports = function(app) {
           result.dataValues.classDate = classDate;
           classes.push(result.dataValues);
         });
-        res.render("classes", { loggedIn: loggedIn, classes: classes });
```
Comment (jkaho) [inline routes/html-routes.js:60]: Add this back too

Comment (jkaho) [reply routes/html-routes.js:60]: line 60

File: routes/html-routes.js (hunk 1 above)
Comment (jkaho) [reply routes/html-routes.js]: line 35-40

**Review by jkaho — CHANGES_REQUESTED** [review]
Look at my comments for html routes and add that small section of code back in

**Comment by klin4994** [comment]
It can now redirect to login page if booking without logging in.

**Review by kvlin — COMMENTED** [review]
- Cancel bookings from profile pages directly
- Close notification by clicking anywhere on the page, no need to wait

**Review by jkaho — APPROVED** [review]

**Pull request merged**